### PR TITLE
Slides of Event Sourcing

### LINKED ISSUE #34: Event Sourcing
## Talk description
Üblicherweise bildet man Entity-Relationship-Modelle über das Relationale Modell ab. Mit Event Sourcing gibt es aber auch noch einen anderen Ansatz. Bei Axilaris haben wir nun das erste Projekt mit diesem Ereignis getriebenen Datenmodell umgesetzt und können aus der Praxis berichten.
Welche Vor- und Nachteile diese beiden Ansätze untereinander haben werde ich aufzeigen. Dabei gehe ich u.a. auf Themen wie Normalisierung, CQRS und Projektionen ein. 
Tatsächlich spielt am Ende auch die Frage eine zentrale Rolle, welcher Ansatz wo passt.

## Talk length (in Minutes)

- [ ] Lightning talk (20-30 Minutes)
- [x] Main talk (45-60 Minutes)

## Preferred language

- [x] German
- [ ] English

## Talk preparation

Do you need support on preparing your talk by e.g. a practice session or a review of your slides? 
We are happy to help!

- [x] Yes, ...
- [ ] No, thanks.

## Preferred date/time

- ...

## Speaker names

- Ronny Hartenstein

## Speakers info

- Axilaris GmbH, Teamleiter Anwendungsentwicklung

## Twitter handle(s)

- @rhflow_de

## Other links

- *nope*

### PR DESCRIPTION
**Pull request by ronnyhartenstein**
Fixes #34

### CODE REVIEW

**Comment by martinkutter** [comment]
Please add the title of your talk to the readme.

File: README.md [hunk 1]
```diff
@@ -3,6 +3,11 @@ Place to share presentation material from the monthly Web Developer Meetup in Ch
 
 ## 2019
 
+### July
```
Comment (MorrisJobke) [inline README.md]:
```suggestion
### July
#### Event Sourcing
```

Comment (martinkutter) [reply README.md]: @MorrisJobke perfect!
Unfortunately only @ronnyhartenstein has the right to commit this suggestion to his branch.

Comment (MorrisJobke) [reply README.md]: He also has the option to grant write permission for this branch when the PR was opened (there is a little checkbox) ... just for the future. 😉

Comment (ronnyhartenstein) [reply README.md]: Waah, sorry, I forgot about that. Hold on a minute..

**Pull request merged**